gapic/src/main/com/google/gapid/GraphicsTraceView.java

```diff
@@ -104,9 +108,28 @@ public void store(TabArea.FolderInfo[] folders) {
 
     tabs.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true));
 
+    @SuppressWarnings("synthetic-access")
```

Comment:
Just make the `syncTabMenuItem` method protected instead of ignoring the warning.

Reply:
Sure, done.